test/datalog_tests/function.fail.dl

```diff
@@ -20,21 +20,6 @@ function vars(): () = {
 
 //---
 
-typedef Alt = C0{x: bit<32>}
-            | C1{x: bit<32>}
-
-// shadow argument name
-function shadow(v: string): () = {
-    var a: Alt = C0{1};
-
-    var i: bit<32> = match (a) {
-        C0{.x = v} -> v,
-        C1{v} -> v
```

Comment:
This looks quite ambiguous. It would be nicer if you required a `var` to always be used when introducing a new var. If you just use an identifier this could be an error.